include/vapor/ParamsMgr.h

```diff
@@ -508,6 +508,12 @@ class PARAMS_API ParamsMgr : public MyBase {
  }
 
  bool GetSaveStateEnabled() const { return (_ssave.GetEnabled()); }
+    
```

Comment:
Please document {Get,Set}SaveStateUndoEnabled AND {Get,Set}SaveStateEnabled so we have a clear understanding of what these closely related methods do and when you might use one versus the other.

Reply:
I documented {Get,Set}SaveStateUndoEnabled.

I did not write the {Get,Set}SaveStateEnabled functions and I don’t know what they are supposed to do. Based on the resulting behavior I am assuming their functionality is broken.

Reply:
Well then let's either fix the older function or replace it, But let's not have two methods that are ostensibly supposed to provide the same functionality but for some reason are not.

Reply:
Well I can’t fix it if I don’t know what it’s supposed to do. For the same reason I also didn’t know that they are supposed to have the same functionality. I can look into whether we can simply remove/replace the old function since it is broken anyway but since it is so old there may be code that depends on its odd behavior.

Reply:
It seems to be used in multiple places and based on the context in which it is used I don't think my new function provides the intended functionality of the old function.

Reply:
I'm going to approve this, but I've opened a ticket to address (refactor) some of the uglyness that is going on with state saving: #2444